src/main/resources/application.properties

```diff
@@ -53,8 +53,8 @@ server.ssl.keyStoreType=PKCS12
 server.ssl.keyAlias=tomcatORWHATEVERRYOUCALLEDIT
 security.headers.frame=false
 
-oicd.privatekey=-----BEGIN PRIVATE KEY-----HERE THE RSA PRIVATE KEY-----END PRIVATE KEY-----
-oicd.publickey=-----BEGIN PUBLIC KEY-----HERE THE RSA PUBLIC KEY-----END PUBLIC KEY-----
+oidc.privatekey=-----BEGIN PRIVATE KEY-----HERE THE RSA PRIVATE KEY-----END PRIVATE KEY-----
```

Comment:
Aren't these used in some place in the code?  Surely the name of the variable will need to change too.

Reply:
Yeah, it is true!! So we are good to go!